Makefile

```diff
@@ -0,0 +1,78 @@
+BASE_DOMAIN=origin-gce.dev.openshift.com
+MOUNT_FLAGS=:z
+PODMAN=sudo podman
+PODMAN_RUN=${PODMAN} run --privileged --rm -v `pwd`:/output${MOUNT_FLAGS} --user `id -u`
```

Comment:
You could change the backticks to $(shell <command>) eg $(shell id -u). Then the evaluated expression is written to stdout. This is just a suggestion I came across while debugging the comment below.

Reply:
Right, good idea